app/core/models/issue.py

```diff
@@ -82,7 +82,7 @@ class CaseOutcome:
 
 
 class IssueManager(models.Manager):
-    def check_permisisons(self, request):
```

Comment:
Want to remove misspellings, aiming for clarity and beauty with the code, so that it is easy to understand and maintain.